auth/src/main/java/org/openedx/auth/presentation/signin/SignInFragment.kt

```diff
@@ -61,41 +56,27 @@ class SignInFragment : Fragment() {
                                 )
 
                                 AuthEvent.ForgotPasswordClick -> {
-                                    viewModel.forgotPasswordClickedEvent()
-                                    router.navigateToRestorePassword(parentFragmentManager)
+                                    viewModel.forgotPasswordClickedEvent(parentFragmentManager)
```

Comment:
IMO, better to update the method names,
- viewModel.navigateToForgotPassword(parentFragmentManager)
- viewModel.navigateToSignUp(parentFragmentManager)
- viewModel.navigateToWhatsNew(parentFragmentManager)

Reply:
I renamed navigateToForgotPassword, and navigateToSignUp, I left proceedWhatsNew as it was, because there is extra logic inside this method